lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -67,6 +67,6 @@ LighthouseReport.prototype = {
   }
 };
 
-(function() {
+window.addEventListener('DOMContentLoaded', _ => {
   return new LighthouseReport();
```

Comment:
hah. wish I could. :)  

![image](https://cloud.githubusercontent.com/assets/39191/20034576/ac0564c0-a37f-11e6-84db-6b39e4c40583.png)

looks like this error doesnt exist in latest eslint/eslint-config-google, so i added a note to tweak once we upgrade.